reana_client/cli/secrets.py

```diff
@@ -53,6 +54,7 @@ def secrets_group(ctx):
     default=False,
     help='Overwrite the secret if already present')
 @add_access_token_options
+@check_connection
```

Comment:
Same as before, moving decorators to `secrets_group`? This applies to all commands that belong to a Click group and all of them they need the check.